react/hooks/useFacetNavigation.js

```diff
@@ -191,6 +191,10 @@ const useFacetNavigation = selectedFacets => {
         selectedFacets
       )
 
+      if (scrollToTop) {
+        window.scrollTo(0, 0)
```

Comment:
Use the smooth-scroll polyfill http://iamdustan.com/smoothscroll/

Then we can use `behavior: 'smooth'`

Reply:
fixed! I used `smooth` in [this workspace](https://thalyta--eriksbikeshop.myvtex.com/bike?_q=bike&map=ft). Thank you!!